lib/receptor_controller/client/response_worker.rb

```diff
@@ -120,12 +119,13 @@ def process_message(message)
             #
             registered_messages.delete(message_id)
 
-            logger.debug("Receptor response: ERROR | message #{message_id} (#{response})")
+            logger.error("Receptor response: ERROR | message #{message_id} (#{response})")
 
             callbacks[:receiver].send(callbacks[:error_callback], message_id, response['code'], response['payload'])
           end
-        else
+        elsif ENV["LOG_ALL_RECEPTOR_MESSAGES"] == "true"
```

Comment:
I don't like testing against string "true" much...

Reply:
I changed it to integers in this PR: https://github.com/RedHatInsights/receptor_controller-client-ruby/pull/17

:)